toto.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+echo hello
+echo world
+
+exit 9
```

Comment:
What does this do?

Reply:
Woops some tests I did to have notification when tests ends, accidentally commited traces of it...
removed